### PR TITLE
AI now actually get the station's frequencies

### PR DESCRIPTION
**Pull request by ShiftyRail**
[bugfix]
I hate the spawn(1) too, but the code related to AI's spawn is very intricate and I would like to keep my sanity this week-end.

### CODE REVIEW

**Comment by jknpj** [comment]
>title says AI
>adds it for borgs too
>"bugfix"

**Comment by ShiftyRail** [comment]
Borgs were also supposed to have them too in the OG PR.

File: code/modules/mob/living/silicon/ai/login.dm [hunk 1]
```diff
@@ -19,4 +19,10 @@
 			O.mode = 1
 			O.emotion = "Neutral"
 	view_core()
+	if (mind && !stored_freqs)
+		to_chat(world, "mind is [mind.key]")
```
Comment (DamianX) [inline code/modules/mob/living/silicon/ai/login.dm]: sieg

Comment (Kurfursten) [reply code/modules/mob/living/silicon/ai/login.dm]: please remove testing string

**Comment by ShiftyRail** [comment]
Hitler removed

**Pull request merged**